compiler-rt/lib/profile/InstrProfiling.h

```diff
@@ -261,6 +261,9 @@ uint64_t __llvm_profile_get_magic(void);
 /*! \brief Get the version of the file format. */
 uint64_t __llvm_profile_get_version(void);
 
+/*! \brief If the binary is compiled with profile correlation. */
+int hasCorrelation();
```

Comment:
The name of this function doesn't follow the existing convention, it should be named `__llvm_profile_has_correlation`.

Reply:
Renamed to `__llvm_profile_has_correlation` at https://github.com/llvm/llvm-project/commit/7fa9930847bbef4319c2d2e9c782eb5c8e6b1892